src/views/GovernanceTaskList/SideNavActions/index.tsx

```diff
@@ -17,9 +18,13 @@ const SideNavActions = () => {
       </div>
       <div className="grid-col margin-top-5">
         <h4>Related Content</h4>
-        <a href="/governance-overview" target="_blank">
+        <UswdsLink
+          href="/governance-overview"
+          variant="external"
```

Comment:
I think this shouldn't be external, the variant should just be unset.

Reply:
So I need to double check with Sai on this. The "external" icon is usually indicates that a link is going to open in a new tab/window; not necessarily an external URL (as the naming implies).

I believe we decided that having the `(opens in a new tab)` and the icon would be the happy path forward. I will double check then merge or make changes accordingly.

Reply:
Yep, that's what we agreed on for 508, the icon and the text.